### PR TITLE
ports two diseases from beestation

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
ports two disease from bee
wizarditis which is the same as the impossible to get one but now achieveable with stable uranium gel
and irridiant cells which is a virus that rapidaly send rads everywhere and if it is combined with rad resonance it could be a great combo obtainabile with stable uranium gel

im a fucking idiot
#### Changelog

:cl:  
rscadd: ported two things as im a lazy asshole who cant code
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
fuck

**Comment by boodaliboo** [comment]
i also have no idea how to fix either bug

File: code/datums/diseases/advance/symptoms/wizarditis.dm [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	else
+		var/mob/living/carbon/H = A.affected_mob
+		if(prob(chance))
+			var/obj/item/staff/S = new(H)
+			if(!H.put_in_hands(S))
+				qdel(S)
+
+
+/datum/symptom/wizarditis/proc/teleport(datum/disease/advance/A)
+	var/turf/L = get_safe_random_station_turf()
```
Comment (missatessatessy) [inline code/datums/diseases/advance/symptoms/wizarditis.dm]: var/turf/L = find_safe_turf()

Comment (missatessatessy) [reply code/datums/diseases/advance/symptoms/wizarditis.dm]: Also revert last commit

Comment (boodaliboo) [reply code/datums/diseases/advance/symptoms/wizarditis.dm]: where?

Comment (boodaliboo) [reply code/datums/diseases/advance/symptoms/wizarditis.dm]: i think i did it

File: code/datums/diseases/advance/symptoms/radiation.dm [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/datum/symptom/radiation
+	name = "Iraddiant Cells"
```
Comment (nesisknesis) [inline code/datums/diseases/advance/symptoms/radiation.dm]: better name. Irradiant is to emit a bright light
```suggestion
	name = "Ionising Cellular Emission"
```

**Review by nesisknesis — CHANGES_REQUESTED** [review]

File: code/datums/diseases/advance/symptoms/radiation.dm [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	desc = "Causes the cells in the host's body to give off harmful radiation."
+	stealth = -3
+	resistance = 2
+	stage_speed = -1
+	transmittable = -1
+	level = 8
```
Comment (nesisknesis) [inline code/datums/diseases/advance/symptoms/radiation.dm]:
```suggestion
	level = 6
```
8 is for super-healing symptoms. Wouldn't even regard as this a 7.

File: code/datums/diseases/advance/symptoms/radiation.dm [hunk 4]
```diff
@@ -0,0 +1,51 @@
+/datum/symptom/radiation
+	name = "Iraddiant Cells"
+	desc = "Causes the cells in the host's body to give off harmful radiation."
```
Comment (nesisknesis) [inline code/datums/diseases/advance/symptoms/radiation.dm]:
```suggestion
	desc = "Causes the host's cells to emit ionising radiation."
```

**Comment by nesisknesis** [comment]
Remove wizarditis as a symptom, thats retarded.

DAS kind of invalidates this but could be used with inorganic biology as a radiation-resistant race to irradiate people around you silently.

**Comment by Ratvar** [comment]
I don't think radiation is a good mechanic that should have more ways to appear

File: code/datums/diseases/advance/symptoms/radiation.dm (hunk 4 above)
Comment (boodaliboo) [reply code/datums/diseases/advance/symptoms/radiation.dm]: eh i guess my thought process was to add more diversity to level 8

**Review by nesisknesis — APPROVED** [review]
cool symptom

**Pull request merged**